regexes/device/mobiles.yml

```diff
@@ -4148,6 +4162,23 @@ Clarmin:
     - regex: 'Clarmin[_ ](B[65]|C1)'
       model: '$1'
 
+# CORN
+CORN:
+  regex: 'CORN'
```

Comment:
`regex: 'CORN '` add space to avoid false positives on ' Cornowser/`

> Mozilla/5.0 (Linux; Android 5.1.1; HWP8max) AppleWebKit/601.2.7 (KHTML, like Gecko) Cornowser/1.0.4.7 Chrome/50.0.2655.1 Mobile Safari/601.2.7

Reply:
Should I open a PR for `Cornowser` browser?

Reply:
> Should I open a PR for `Cornowser` browser?

in a separate pull request